inspirehep/records/config.py

```diff
@@ -447,7 +447,7 @@
         },
         "aggs": {
             "field_of_interest": {
-                "terms": {"field": "arxiv_categories", "missing": "Other"},
+                "terms": {"field": "arxiv_categories", "missing": "Other", "size": 166},
```

Comment:
`500`? number of arXiv categories is not fixed

Reply:
this is number from schema, so you're saying we are likely to add more categories in the future?

Reply:
yes, they regularly add new categories